src/pages/cli/project/troubleshooting.mdx

```diff
@@ -0,0 +1,180 @@
+1.	[Deploying multiple index changes in a single amplify push](https://docs.amplify.aws/cli/graphql/troubleshooting/#deploying-multiple-index-changes-at-once)
+2.	[Backfill OpenSearch index from DynamoDB table](https://docs.amplify.aws/cli/graphql/troubleshooting/#deploying-multiple-index-changes-at-once)
+3.	[Index with Multi-Sort Key Fields](https://docs.amplify.aws/cli/graphql/troubleshooting/#index-with-multiple-sort-key-fields)
+
+## Deployment Best Practices
+This section describes the commonly followed best practices which help developers avoid deployment errors and scaling problems.
+### Overriding Amplify CLI Behavior 
+The Amplify CLI follows a unidirectional data flow. Any changes to your application must be performed only through the Amplify CLI and the associated configuration files or through Amplify Studio. You should never manually edit any resources directly from the AWS console or using the AWS CLI/SDK, unless the Amplify documentation explains that it’s necessary to achieve something. If you wish to override the default behavior of Amplify CLI generated resources, you should follow the instructions in the [Amplify extensibility documentation](https://docs.amplify.aws/cli/#extensibility) which details all the “escape hatches” available in the Amplify CLI framework. 
+
+### Changing Model Names in the GraphQL Schema
+<Callout information>
+Error Messages: “An error occurred during the push operation: Removing a model from the GraphQL schema will also remove the underlying DynamoDB table”, “ALL EXISTING DATA IN THESE TABLES WILL BE LOST!”
+</Callout>
+
+If you've previously pushed the schema to the cloud and then rename the table in the schema, then the subsequent push will fail and require you to pass `—allow-destructive-graphql-schema-updates`. This flag will result in the deletion of the backing dynamo table for the old model and subsequent creation of the tables with the new model names. The exception to this is the new @mapsTo directive which has not released yet, allowing you to preserve data. 
+###### Example 1: The following scenario shows the error displayed by the Amplify CLI when you try to push a schema change to the cloud (for an already deployed table). 
+
+###### fig.1.3 Amplify GraphQL Model Deletion Warning/Error
+![ts-guide-model-deletion-warning](/images/ts-guide-model-deletion-warning.png)
+
+### Migration from GraphQL Transformer V1 to GraphQL Transformer V2
+If you have previously deployed an application using GraphQL Transformer V1 and have decided to migrate to GraphQL transformer V2 , please go through the [GraphQL Transformer migration document](https://docs.amplify.aws/cli/migration/transformer-migration/#changes-that-amplify-cli-will-auto-migrate-for-you) to understand the impact of the migration on the schema directives and limitations prior to updating. 
```

Comment:
For this section, it might be worth calling out specific feature flags that control the transformer version and that the version is printed in `amplify status`.